.circleci/config.yml

```diff
@@ -72,32 +73,35 @@ jobs:
             cd cf-graph
             source activate run_env
             # hack around libarchive being dumb
-            conda-forge-tick --run 3 
+            conda-forge-tick --run 3
       - run:
           name: Stage 4
           when: always
           command: |
             cd cf-graph
             source activate run_env
             # hack around libarchive being dumb
-            conda-forge-tick --run 4 
+            conda-forge-tick --run 4
       - run:
           name: deploy
           when: always
           command: |
             cd cf-graph
             source activate run_env
             # hack around libarchive being dumb
-            conda-forge-tick --run -1  
+            conda-forge-tick --run -1
+      - run:
+          name: trigger the next worker
+          when: always
+          command: |
+            python restart_me.py
+
 workflows:
   version: 2
-  hourly:
-    triggers:
-      - schedule:
-          cron: "00 * * * *"
```

Comment:
Side note can we also have cron jobs (for example for the audit) or will those interfere?

Reply:
Yes we can.